substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVDebugInfo.java

```diff
@@ -53,4 +76,12 @@ public CVTypeSectionImpl getCVTypeSection() {
         return cvTypeSection;
     }
 
+    public byte getHeapbaseRegister() {
+        return heapbaseRegister;
+    }
+
+    @SuppressWarnings("unused")
+    public byte getThreadRegister() {
+        return threadRegister;
+    }
```

Comment:
Given those accessor methods the the javadoc comments on the fields above are redundant.

Reply:
Comments removed.